realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmJsonTypeHelper.java

```diff
@@ -243,6 +243,18 @@ public static void emitFillRealmListFromStream(String interfaceName, String gett
     }
     //@formatter:on
 
+    // @formatter:off
+    public static void emitFillRealmIntegerFromStream(String interfaceName, String getter, String setter, String fieldName, JavaWriter writer) throws IOException {
```

Comment:
I don't think so.  The semantics are pretty different.  If it turns out to be very similar, I will re-combine things.